### ISSUE TITLE
xpra multi-screen shadow has pointer issues

### ISSUE THREAD

**Issue by totaam** [Xpra-org/xpra#2249]
*Issue migrated from trac ticket # 2249*

**component:** client | **priority:** major | **resolution:** fixed

#### 2019-04-03 10:58:39: stdedos created the issue

___
> ```
> Shadow-server:
> 2019-04-03 11:40:49,790 xpra GTK2 shadow version 2.5-[r22096](../commit/7ef6c348c85fc4b162b480215df86cbfa2dba02e) 64-bit
> 2019-04-03 11:40:49,794  running with pid 18889 on Linux Ubuntu 16.04 xenial
> ==========
> Client:
> set "XPRA_EXECUTABLE=C:\Users\stavros.ntentos\Documents\xpra-clients\Xpra-Client-Python3-x86_64_2.5-[r22135](../commit/9632bbcf26034732fde214891f741018f63048a5)"
> 2019-04-03 11:40:51,056 Python/GTK3 Microsoft Windows 10 aero client version 2.5-[r22135](../commit/9632bbcf26034732fde214891f741018f63048a5) 64-bit
> ```
> 
> 
> ```
> Xpra-Client-Python3-x86_64_2.5-[r22135](../commit/9632bbcf26034732fde214891f741018f63048a5)\xpra_cmd" shadow ssh://user@ip/0  --clipboard=no --opengl=no --desktop-scaling=0.75 --webcam=no --speaker=off --microphone=off
> ```
> 
> (splitting from #2131, since it feels that it's worth a bug of it's own)
> 
> There is a weird issue that happens only in multi-screen shadow.
> 
> tl;dr: all shadow client windows "translate mouse coordinates to 0,0 of the X11 screen, not the 0,0 of the X11 monitor". Please check attachments for the visual representation.
> 
> I believe to be a client issue, since the pre-#1801 way seems to work nicely.
> 
> -Also, I can only start `XPRA_SHADOW_MULTI_WINDOW=0 xpra shadow` directly on the server. If I do it inside an `xpra start ssh://user@ip/200 --start=gnome-terminal` it's not working. Is this a bug / do you want a new bug for that?*
